src/modules/flea-market-fee.js

```diff
@@ -1,4 +1,17 @@
 // https://escapefromtarkov.gamepedia.com/Trading#Flea_Market
+const localStorageReadJson = (key, defaultValue) => {
+    try {
+        const value = localStorage.getItem(key);
+
+        if (typeof value === 'string') {
+            return JSON.parse(value);
+        }
+    } catch (error) {
+        /* noop */
```

Comment:
Do we foresee any issues with suppressing errors here?

Reply:
I don't think so - it defaults to returning defaultValue in case of an error. This is the same function we use on another export, just copied over.